addons/portal/static/src/scss/portal.scss

```diff
@@ -13,10 +13,10 @@ $o-portal-mobile-toolbar-border: $border-color;
 $o-portal-mobile-toolbar-bg: $gray-200;
 
 // Portal Tables
-$o-portal-table-th-pt: map-get($spacers, 2) !default; // bts4 pt-2
-$o-portal-table-th-pb: map-get($spacers, 2) !default; // bts4 pb-2
-$o-portal-table-td-pt: map-get($spacers, 1) !default; // bts4 pt-1
-$o-portal-table-td-pb: map-get($spacers, 1) !default; // bts4 pb-1
+$o-portal-table-th-pt: map-get($spacers, 1) !default;
+$o-portal-table-th-pb: map-get($spacers, 1) !default;
+$o-portal-table-td-pt: map-get($spacers, 2) !default; // bts4 pt-2
+$o-portal-table-td-pb: map-get($spacers, 2) !default; // bts4 pb-2
 
 // Portal custom bg color
 $o-portal-bg-color: desaturate($gray-200, 100%);
```

Comment:
This variable (and the following `$o-portal-use-default-colors` one) is probably not needed anymore, could you check? Eventually these lines should be removed.
Don't forget to check if the variables are used by other modules. 

Removing these variables should be in a separate commit with a proper description ;)